docs/pages/2020_Intro_Week/index.rst

```diff
@@ -49,20 +113,15 @@ Appendices
    :maxdepth: 1
    :numbered:
 
-   ./appendices/groups
    ./appendices/exploring_provenance
 
-https://github.com/aiidateam/aiida-tutorials/issues/229
-
-.. _2020:appendix_computer_code_setup:
-
-Setting up computers and codes
-..............................
+.. https://github.com/aiidateam/aiida-tutorials/issues/229
 
 Acknowledgements
 ----------------
 
-The `AiiDA virtual tutorial <http://www.aiida.net/tutorial-reproducible-workflows/>`__  was made possible by support from the MaX European Centre of Excellence, the swissuniversities P-5 project “Materials Cloud”, the MARVEL National Centre of Competence in Research, the H2020 MARKETPLACE project, the H2020 INTERSECT project and the PASC project.
+We are very grateful to the plugin developers for their contributed talks.
+The `AiiDA virtual tutorial <http://www.aiida.net/aiida-virtual-tutorial-july-2020/>`__  was made possible by support from the MaX European Centre of Excellence, the swissuniversities P-5 project “Materials Cloud”, the MARVEL National Centre of Competence in Research, the H2020 MARKETPLACE project, the H2020 INTERSECT project and the PASC project.
```

Comment:
This will make the lines appear directly underneath each other and set apart the thanks to the plugin developers.

This could also be highlighted even more by having an empty line in between, of course. Either is fine with me.

```suggestion
| We are very grateful to the plugin developers for their contributed talks.
| The `AiiDA virtual tutorial <http://www.aiida.net/aiida-virtual-tutorial-july-2020/>`__  was made possible by support from the MaX European Centre of Excellence, the swissuniversities P-5 project “Materials Cloud”, the MARVEL National Centre of Competence in Research, the H2020 MARKETPLACE project, the H2020 INTERSECT project and the PASC project.
```